protocols/raft/src/test/java/io/atomix/protocols/raft/storage/log/AbstractLogTest.java

```diff
@@ -233,15 +209,15 @@ public void testResetTruncateZero() throws Exception {
     RaftLogReader reader = log.openReader(1, RaftLogReader.Mode.ALL);
 
     assertEquals(0, writer.getLastIndex());
-    writer.append(new InitializeEntry(1, System.currentTimeMillis()));
-    writer.append(new InitializeEntry(1, System.currentTimeMillis()));
+    writer.append(ENTRY);
```

Comment:
Does it matter that we no longer increment the second parameter?

Reply:
nope... there was not much point to using real entries. Better to have determinism anyways